src/react/components/pages/editorPage/editorSideBar.tsx

```diff
@@ -102,8 +100,8 @@ export default class EditorSideBar extends React.Component<IEditorSideBarProps,
         this.props.onAssetSelected(asset);
     }
 
-    private rowRenderer = ({ key, index, style }): JSX.Element => {
-        const asset = this.props.assets[index];
+    private rowRenderer = ({ columnIndex, key, rowIndex, style }): JSX.Element => {
```

Comment:
It doesn't look like rowIndex is being used in this method, so you can get rid of it as a parameter.